taxcalc/current_law_policy.json

```diff
@@ -2293,13 +2363,15 @@
         "row_label": ["2013",
                       "2014",
                       "2015",
-                      "2016"],
+                      "2016",
+                      "2017"],
         "cpi_inflated": false,
```

Comment:
@MattHJensen, yes. I found this line in a [CBO article](https://www.cbo.gov/budget-options/2016/52264) that indicates this should be `'cpu_inflated': true`.
>This option would lower the threshold for the EITC investment income test from $3,400 to $1,700. As under current law, that threshold would be indexed to include the effects of inflation.

I'll make the change and submit a new commit.